Packs/GCP-Enrichment-Remediation/Playbooks/playbook-Cloud_Credentials_Rotation_-_GCP.yml

```
@@ -0,0 +1,805 @@
+      - - operator: isEqualString
+          left:
+            value:
+              complex:
+                root: inputs.identityType
+            iscontext: true
+          right:
+            value:
+              simple: SERVICE_ACCOUNT
+      - - operator: isNotEmpty
+          left:
+            value:
+              complex:
+                root: inputs.zone
+            iscontext: true
+      - - operator: isNotEmpty
+          left:
+            value:
+              complex:
+                root: inputs.serviceAccountEmail
+            iscontext: true
+    - label: USER
+      condition:
+      - - operator: isEqualString
+          left:
+            value:
+              complex:
+                root: inputs.identityType
+            iscontext: true
+          right:
+            value:
+              simple: User
+          ignorecase: true
+      - - operator: isNotEmpty
+          left:
+            value:
+              complex:
+                root: inputs.userID
+            iscontext: true
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 1440,
+          "y": 110
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+view: |-
+  {
+    "linkLabelsPosition": {
+      "14_11_Revoke Access Key": 0.77,
+      "14_26_Reset Password": 0.5,
+      "28_10_USER": 0.73,
+      "28_18_SA": 0.68
+    },
+    "paper": {
+      "dimensions": {
+        "height": 1655,
+        "width": 1970,
+        "x": 890,
+        "y": -40
+      }
+    }
+  }
+inputs:
+- key: GSuiteRemediationType
+  value: {}
+  required: false
+  description: |-
+    The response playbook provides the following remediation actions using GSuite Admin:
+
+    Reset: By entering "Reset" in the input, the playbook will execute password reset.
+
+    Revoke: By entering "Revoke" in the input, the playbook will execute access key suspension. (Both the user and client IDs must be provided)
+
+    ALL: By entering "ALL" in the input, the playbook will execute the password reset and revoke access key tasks.
+  playbookInputQuery:
+- key: userID
+  value: {}
+  required: false
+  description: Identifies the user in the API request. The value can be the user's primary email address, alias email address, or unique user ID.
+  playbookInputQuery:
+- key: clientID
+  value: {}
+  required: false
+  description: The client ID.
+  playbookInputQuery:
+- key: zone
+  value: {}
+  required: false
+  description: The name of the zone.
+  playbookInputQuery:
+- key: serviceAccountEmail
+  value: {}
+  required: false
+  description: The service account email.
+  playbookInputQuery:
+- key: identityType
+  value: {}
+  required: false
+  description: |-
+    The type of identity involved. Usually mapped to incident field named 'cloudidentitytype'.
+    e.g.
+    USER, SERVICE_ACCOUNT,APPLICATION
```

```suggestion
    e.g.,
    USER, SERVICE_ACCOUNT,APPLICATION
```
should there be a space in between entities?
After USER, there is a space
After SERVICE_ACCOUNT, there is no space